REVIEW.md

```diff
@@ -0,0 +1,27 @@
+# Review Guidelines
+
+Our review guidelines are intended to provide clear steps for PR proposers and reviewers.
+
+Only published PRs will be considered for review. Draft PRs will be considered in-progress and not yet ready for review.
+
+## Rules
```

Comment:
This was a list I wrote down quickly in some minutes.. I think this could get worked out to a real document.. idk :-) 

btw..: Thanks Ryan for taking those drafts to the next step! 💪